### PR TITLE
Homework7: Pavel Mitrafanau

### PR DESCRIPTION
**Pull request by Pavel2812mi**
Hello, Sergei 
This homework was an uneasy one, but I think I did it 
Please check if everything is correct

### CODE REVIEW

File: Homework7.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    cows = 0
+    bulls = 0
+
+    # Проверяем на 'коров' (совпадение цифр)
+    for i in range(len(str(number))):
+        if str(number)[i] in str(random_number):
+            cows += 1
+
+    # Проверяем на 'быков' (совпадение цифр и их позиций)
+    for i in range(len(str(number))):
```
Comment (Stml89) [inline Homework7.py:29]: Calculations does not work correctly, try:
random number:3649
guess: 3694

File: Homework7.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    random_number = random.randint(1000, 9999)
+    # Проверяем, что все 4 цифры уникальны
+    if len(str(random_number)) == len(set(str(random_number))):
+        print(random_number)   # Выводим число для отладки (можно убрать)
+        break
+print("Загадано")
+
+# Начинаем цикл угадывания (загадонное число при этом не меняется)
+while True:
+    number = int(input("Введите ваше число: "))
```
Comment (Stml89) [inline Homework7.py]: it is possible to enter non unique number

**Review by Stml89 — CHANGES_REQUESTED** [review]

**Comment by Pavel2812mi** [comment]
Code fixed, please review again

File: Homework7.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
+"""
+Homework_7
+"""
+# pylint: disable=invalid-name
```
Comment (Stml89) [inline Homework7.py:3]: remove it

File: Homework7.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+import random
+while True:
+    random_number = random.randint(1000, 9999)
+    # Проверяем, что все 4 цифры уникальны
+    if len(str(random_number)) == len(set(str(random_number))):
+        print(random_number)   # Выводим число для отладки (можно убрать)
+        break
+print("Загадано")
+
+# Начинаем цикл угадывания (загаданное число при этом не меняется)
```
Comment (Stml89) [inline Homework7.py]: do not need to comment obvious actions

**Review by Stml89 — COMMENTED** [review]
You can merge it

**Pull request merged**